commitlint.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+   rules: {
+      'body-leading-blank': [ 2, 'always' ],
```

Comment:
As team decided on Slack: add `body-max-line-length` of `90`

Reply:
For posterity: this matches our [comments max line length](https://github.com/silvermine/eslint-config-silvermine/blob/96dd9215a4fa7514454396729753b5ce97c04b01/index.js#L157)

Reply:
And thus, let's also add `footer-max-line-length` of `90`